README.textile

```diff
@@ -152,3 +152,5 @@ Legend:
 [-] Bug fixed (we hope)
 [!] Important info
 </pre>
+
+!{width: 10%; height: 10%}https://compteur-visites.ennder.fr/sites/31/token/githubcip/image!
```

Comment:
Is this change needed for this pull request?

Reply:
Not at all, it's something I use to see if my plugin has some success.

Reply:
Thanks.
I'm not a maintainer of this repository but I think that not related changes should be done in separated pull request in general terms.

Reply:
Hello kou.
You're right, I'm going to remove the image link.

Reply:
Done.

Reply:
Thanks!